core/docker-entrypoint.sh

```diff
@@ -4,12 +4,62 @@ export QDB_PACKAGE=${QDB_PACKAGE:-docker}
 
 QUESTDB_DATA_DIR=${QUESTDB_DATA_DIR:-"/var/lib/questdb"}
 IGNORE_DATA_ROOT_MOUNT_CHECK=${IGNORE_DATA_ROOT_MOUNT_CHECK:-"false"}
-IGNORE_FIND_AND_OWN_DIR=${IGNORE_FIND_AND_OWN_DIR:-"false"}
+FIND_AND_OWN_DIR=${FIND_AND_OWN_DIR:-"false"}
```

Comment:
`DO_CHOWN` may be a better name?

Reply:
we also have `FIND_AND_OWN_DIR`, maybe i should move it before i call `find_and_own_dir` and not inside the function